packages/itwin/grouping-mapping-widget/src/widget/components/utils.tsx

```diff
@@ -152,3 +153,15 @@ export const LoadingSpinner = () => {
     </div>
   );
 };
+
+const prefixUrl = (baseUrl?: string, prefix?: string) => {
+  if (prefix && baseUrl) {
+    return baseUrl.replace("api.bentley.com", `${prefix}api.bentley.com`);
+  }
+  return baseUrl;
+};
+
+export const getReportingClient = (prefix?: "" | "qa" | "dev") => {
+  const url = prefixUrl(REPORTING_BASE_PATH, prefix ?? process.env.IMJS_URL_PREFIX);
```

Comment:
Remove IMJS_URL_PREFIX as we are not planning on supporting that.

Reply:
Removed